src/compositor/view.c

```diff
@@ -2,6 +2,7 @@
 #include <stdio.h>
 #include <string.h>
 #include <assert.h>
+#include <sys/types.h>
```

Comment:
This already comes from view.h that is included above. So probably not needed. It's needed in header since pid_t is exposed.